src/modules/rc_update/parameters.cpp

```diff
@@ -1,325 +0,0 @@
-
-	parameter_handles.rc_map_aux1 = param_find("RC_MAP_AUX1");
-	parameter_handles.rc_map_aux2 = param_find("RC_MAP_AUX2");
-	parameter_handles.rc_map_aux3 = param_find("RC_MAP_AUX3");
-	parameter_handles.rc_map_aux4 = param_find("RC_MAP_AUX4");
-	parameter_handles.rc_map_aux5 = param_find("RC_MAP_AUX5");
-	parameter_handles.rc_map_aux6 = param_find("RC_MAP_AUX6");
-
-	/* RC to parameter mapping for changing parameters with RC */
-	for (int i = 0; i < rc_parameter_map_s::RC_PARAM_MAP_NCHAN; i++) {
-		char name[rc_parameter_map_s::PARAM_ID_LEN];
-		snprintf(name, rc_parameter_map_s::PARAM_ID_LEN, "RC_MAP_PARAM%d",
-			 i + 1); // shifted by 1 because param name starts at 1
-		parameter_handles.rc_map_param[i] = param_find(name);
-	}
-
-	parameter_handles.rc_map_flightmode = param_find("RC_MAP_FLTMODE");
-
-	/* RC thresholds */
-	parameter_handles.rc_fails_thr = param_find("RC_FAILS_THR");
-	parameter_handles.rc_assist_th = param_find("RC_ASSIST_TH");
-	parameter_handles.rc_auto_th = param_find("RC_AUTO_TH");
-	parameter_handles.rc_rattitude_th = param_find("RC_RATT_TH");
-	parameter_handles.rc_posctl_th = param_find("RC_POSCTL_TH");
-	parameter_handles.rc_return_th = param_find("RC_RETURN_TH");
-	parameter_handles.rc_loiter_th = param_find("RC_LOITER_TH");
-	parameter_handles.rc_acro_th = param_find("RC_ACRO_TH");
-	parameter_handles.rc_offboard_th = param_find("RC_OFFB_TH");
-	parameter_handles.rc_killswitch_th = param_find("RC_KILLSWITCH_TH");
-	parameter_handles.rc_armswitch_th = param_find("RC_ARMSWITCH_TH");
-	parameter_handles.rc_trans_th = param_find("RC_TRANS_TH");
-	parameter_handles.rc_gear_th = param_find("RC_GEAR_TH");
-	parameter_handles.rc_stab_th = param_find("RC_STAB_TH");
-	parameter_handles.rc_man_th = param_find("RC_MAN_TH");
-
-	/* RC low pass filter configuration */
-	parameter_handles.rc_flt_smp_rate = param_find("RC_FLT_SMP_RATE");
-	parameter_handles.rc_flt_cutoff = param_find("RC_FLT_CUTOFF");
-
-	// These are parameters for which QGroundControl always expects to be returned in a list request.
-	// We do a param_find here to force them into the list.
-	(void)param_find("RC_CHAN_CNT");
-}
-
-int update_parameters(const ParameterHandles &parameter_handles, Parameters &parameters)
-{
-	bool rc_valid = true;
-	float tmpScaleFactor = 0.0f;
-	float tmpRevFactor = 0.0f;
-	int ret = PX4_OK;
-
-	/* rc values */
-	for (unsigned int i = 0; i < RC_MAX_CHAN_COUNT; i++) {
-
-		param_get(parameter_handles.min[i], &(parameters.min[i]));
-		param_get(parameter_handles.trim[i], &(parameters.trim[i]));
-		param_get(parameter_handles.max[i], &(parameters.max[i]));
-		param_get(parameter_handles.rev[i], &(parameters.rev[i]));
-		param_get(parameter_handles.dz[i], &(parameters.dz[i]));
-
-		tmpScaleFactor = (1.0f / ((parameters.max[i] - parameters.min[i]) / 2.0f) * parameters.rev[i]);
-		tmpRevFactor = tmpScaleFactor * parameters.rev[i];
-
-		/* handle blowup in the scaling factor calculation */
```

Comment:
You did not want to keep that check?

Reply:
I'm not opposed to keeping it, but not in this form printing an error message that's likely never going to be seen by a user. The scaling_factor it clears isn't even used anymore.